setup.cfg

```
@@ -22,8 +22,8 @@ packages =
     lavalink
 python_requires = >=3.5.3
 install_requires =
-    websockets>=8.1
-    discord.py>=1.2.5
+    aiohttp==3.6.2
```

Please use `>=` so that we don't have to update RLL each time we bump aiohttp dep in Red
```suggestion
    aiohttp>=3.6.2
```